src/main/java/Main.java

```diff
@@ -0,0 +1,16 @@
+import core.basesyntax.Figure;
+import java.util.Random;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Figure[] shape = new Figure[new Random().nextInt(15)];
```

Comment:
15 is a magic number
```suggestion
        Figure[] figures = new Figure[new Random().nextInt(15)];
```